egui/src/widgets/plot.rs

```diff
@@ -89,12 +97,34 @@ impl Bounds {
         self.max[1] += pad;
     }
 
-    pub fn union_mut(&mut self, other: &Bounds) {
+    pub fn merge(&mut self, other: &Bounds) {
         self.min[0] = self.min[0].min(other.min[0]);
         self.min[1] = self.min[1].min(other.min[1]);
         self.max[0] = self.max[0].max(other.max[0]);
         self.max[1] = self.max[1].max(other.max[1]);
     }
+
+    pub fn shift_x(&mut self, delta: f64) {
```

Comment:
I think [`translate`](https://en.wikipedia.org/wiki/Translation_(geometry)) is a more common word for this

Reply:
Changed in b3002ff

Reply:
Changed in b3002ff